tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -3660,6 +3674,15 @@ private function setUserDisplayName(string $user): void {
 		$this->setCurrentUser($currentUser);
 	}
 
+	private function setUserEmail(string $user): void {
+		$currentUser = $this->setCurrentUser('admin');
+		$this->sendRequest('PUT', '/cloud/users/' . $user, [
+			'key' => 'email',
+			'value' => $user . '@domain.tld'
```

Comment:
*nit:* Lets use the value we are supposed to use:
```suggestion
			'value' => $user . '@example.tld'
```